## Esercizi - Formattazione per l'output

<b>Esercizio 1</b>

Stampa la frase <code>La mela è </code> seguita dalla parola <code>rossa</code>.<br>
La parola <code>rossa</code> aggiungila con il format, in versione estesa.<br>
Output desiderato:<br>
<code><font color=green>**La mela è rossa**</font></code>

In [1]:
print('La mela è {}'.format('rossa'))

La mela è rossa


<b>Esercizio 2</b>

Avendo questo <code>format('verdura','frutta')</code> ottieni questo risultato:<br>
Output desiderato:<br>
<code><font color=green>**Preferisco la frutta alla verdura**</font></code><br>
Utilizza gli indici.

In [2]:
print('Preferisco la {1} alla {0}'.format('verdura','frutta'))

Preferisco la frutta alla verdura


<b>Esercizio 3</b>

Avendo questo <code>format(professione='studente',nome='Olivia',eta='24')</code> ottieni questo risultato:<br>
Output desiderato:<br>
<code><font color=green>**Il mio nome è Olivia, ho 24 anni e sono una studente universitaria.**</font></code><br>
Utilizza le parole chiave per stampare l'output.

In [3]:
print('Il mio nome è {nome}, ho {eta} anni e sono una {professione} universitaria.'.format(professione='studente',nome='Olivia',eta='24'))

Il mio nome è Olivia, ho 24 anni e sono una studente universitaria.


<b>Esercizio 4</b>

In [4]:
nome = 'Giovanni'
cognome = 'Bianchi'

Ottieni questo output utilizzando le variabili sopra indicate e il format in forma abbreviata.<br>
Output desiderato:<br>
<code><font color=green>**Il mio nome è Giovanni e il mio cognome è Bianchi.**</font></code><br>

In [5]:
print(f'Il mio nome è {nome} e il mio cognome è {cognome}.')

Il mio nome è Giovanni e il mio cognome è Bianchi.


<b>Esercizio 5</b>

Stampa esattamente questo risultato:<br>
Output desiderato:<br>
<code><font color=green>**Il sole \t e la luna.**</font></code><br>

In [6]:
print(r'Il sole \t e la luna.')

Il sole \t e la luna.


<b>Esercizio 6</b>

Stampa il numero <code>10.57523</code>, utilizza il format. <br>
Output desiderato:<br>
<code><font color=green>**Il numero è 10.57523**</font></code><br>

In [7]:
print('Il numero è {}'.format(10.57523))

Il numero è 10.57523


<b>Esercizio 7</b>

Stampa il numero <code>10.57523</code> arrotondato a due decimali, utilizza il format. <br>
Output desiderato:<br>
<code><font color=green>**Il numero è 10.58**</font></code><br>

In [8]:
print('Il numero è {valore:1.2f}'.format(valore=10.57523))

Il numero è 10.58
